hieradata_aws/common.yaml

```diff
@@ -468,6 +468,8 @@ govuk::apps::ckan::s3_aws_secret_access_key: "%{hiera('s3_aws_secret_access_key'
 
 govuk::apps::collections::unicorn_worker_processes: 8
 
+# TODO: switch to "collections-publisher-mysql" and uncomment the 'push'
+# `govuk_env_sync::tasks` tasks when we're ready to switch to the dedicated RDS instance
 govuk::apps::collections_publisher::db_hostname: "mysql-primary"
 govuk::apps::collections_publisher::jwt_auth_secret: "%{hiera('jwt_auth_secret')}"
 govuk::apps::collections_publisher::redis_host: "%{hiera('sidekiq_host')}"
```

Comment:
Unlike the PostgreSQL equivalent, we _don't_ need to set a `s_collections_publisher_db_admin::apt_mirror_hostname` [here](https://github.com/alphagov/govuk-puppet/blob/10871f283d3e8c1c848d89dbba7610b52f5669e6/hieradata_aws/common.yaml#L862), as it isn't used anywhere in the `govuk::node::s_collections_publisher_db_admin` class.